Content.Server/Fluids/Components/BucketComponent.cs

```diff
@@ -30,6 +31,14 @@ public class BucketComponent : Component, IInteractUsing
 
         private List<EntityUid> _currentlyUsing = new();
 
+        public Solution? BucketSolution
+        {
+            get
+            {
+                EntitySystem.Get<SolutionContainerSystem>().TryGetSolution(Owner, SolutionName, out var solution);
+                return solution;
+            }
+        }
```

Comment:
im sure there are other comps that do this, but it's preferable to just get the solution directly rather than having a property for it

Reply:
new push should fix this (unless we misunderstood the assignment)